beater/sipcmbeat.go

```diff
@@ -315,9 +316,21 @@ func (bt *Sipcmbeat) publishEv(srcEv *calltr.EventData) {
 	}
 	addFields(event.Fields, "event.call_start", ed.StartTS)
 	addFields(event.Fields, "client.transport", ed.ProtoF.ProtoName())
-	addFields(event.Fields, "client.ip", ed.Src)
+	if c, err := anonymization.EncryptIP(bt.anonymKey, ed.Src); err != nil {
```

Comment:
We need  to check if encryption is enabled (a  new config opion or 2, something like  bt.Config.Encrypt = ENC_IPs | ENC_URIs | ENC_CallIds.... +  bt.Config.EncKey).